publications.json

```diff
@@ -15644,6 +15644,19 @@
     "url": "https://openmobilealliance.org/documents/whitepapers/OMA-WP-LwM2M-for-Utilities/OMA-WP-LwM2M-Utilities-Benefits-20231001-A.pdf",
     "description": "Efficiency, Security, and Cost Savings Through OMA LwM2M In an era characterized by relentless technological advancement and an ever-expanding network of connected devices, the utility sector faces a pressing need for innovation. To meet the challenges of today's utilities ecosystem, the OMA LightweightM2M Protocol emerges as a beacon of hope, promising to reshape the landscape of Field Area Management and unlock a myriad of benefits for utilities.",
     "contentMediaType": "application/pdf"
+  },
+   {
+    "organization": "https://technical.openmobilealliance.org/organization-info.json",
+    "enablerName": "OMA LightweightM2M (LwM2M)",
+    "enablerAbbreviation": "LightweightM2M",
+    "status": "Approved",
+    "version": "",
+    "releaseDate": "2023-11-14",
+    "fileName": "OMA-WP-LwM2M-Metering-Use-Case-20231001-A",
+    "docType": "Whitepaper",
+    "url": "https://openmobilealliance.org/documents/whitepapers/OMA-WP-LwM2M-for-Utilities/OMA-WP-LwM2M-Utilities-Benefits-20231001-A.pdf",
```

Comment:
@gocadimic, is this the right document? I think that it should be the Metering Use Case. Please have a look. Thanks

Reply:
@jpradocueva, 
Yes, you are right; the URL wasn't correct. It's rectified now.